pkg/build/controller/buildpod/controller.go

```diff
@@ -64,13 +68,19 @@ func NewBuildPodController(buildInformer cache.SharedIndexInformer, podInformer
 	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: v1core.New(extkc.Core().RESTClient()).Events("")})
 
 	buildListerUpdater := buildclient.NewOSClientBuildClient(oc)
+	buildConfigGetter := buildclient.NewOSClientBuildConfigClient(oc)
+	buildDeleter := pruner.NewBuildDeleter(oc)
```

Comment:
refactor BuildDeleter out of the prune backage and into where the other build clients live.